dist/icon-activity.css

```diff
@@ -2,8 +2,16 @@
   background-image: url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2223%22%20height%3D%2223%22%20viewBox%3D%220%200%2023%2023%22%3E%3Ctitle%3Eicons-singletons%3C%2Ftitle%3E%3Cpath%20d%3D%22M11.31%2C11.1L9.2%2C1.56a0.94%2C0.94%2C0%2C0%2C0-1.82%2C0L4.5%2C12.77H1.31a0.93%2C0.93%2C0%2C0%2C0%2C0%2C1.87H5.23a0.93%2C0.93%2C0%2C0%2C0%2C.91-0.7L8.22%2C5.78l1.53%2C6.88Z%22%20fill%3D%22%23375667%22%2F%3E%3Cpath%20d%3D%22M21.68%2C12.15H18.09L16%2C7.87a0.94%2C0.94%2C0%2C0%2C0-1.76.22l-1.71%2C8.6-1-4.44L10%2C13.82l1.69%2C7.6a0.94%2C0.94%2C0%2C0%2C0%2C1.83%2C0l2-10.2%2C1.1%2C2.29a0.94%2C0.94%2C0%2C0%2C0%2C.84.53h4.18A0.94%2C0.94%2C0%2C0%2C0%2C21.68%2C12.15Z%22%20fill%3D%22%23375667%22%2F%3E%3C%2Fsvg%3E');
```

Comment:
My bad - it looks like it rebuilt the icon CSS by just adding the CSS to the existing files. Instead of running that specific `css-svg-build` task, it looks like we should be running the `build-icons` task:

grunt.registerTask('build-icons', ['clean:pre_svg', 'grunt-svg-css', 'copy:svg', 'concat:svg', 'clean:svg']);

Reply:
Gotcha.  At this point I wonder if I can still do this considering everything is in dist/ now?  I'll try it out, if I manually copy everything back.

Reply:
Update, I can, but the `build-icons` must be responsible for porting everything over to dist as well.